src/service/data_products/genome_attributes.py

```diff
@@ -454,29 +453,38 @@ async def _get_internal_match_id(
 
 async def _perform_filter(store: ArangoStorage, filters: FilterSet, output_table: bool):
     aql, bind_vars = filters.to_arangosearch_aql()
-    doc = None
-    data = []
     cur = await store.execute_aql(aql, bind_vars=bind_vars)
     try:
-        async for doc in cur:
-            # ?ODO FiLTERS this code is pretty similar to the query_table code. See if it can be
-            # merged once everything's functional
-            doc = _remove_keys(doc)
-            if output_table:
-                data.append([doc[k] for k in sorted(doc)])
-            else:
-                data.append({k: doc[k] for k in sorted(doc)})
+        if filters.count:
+            count = await cur.next()
+            return {
+                _FLD_SKIP: 0,
+                _FLD_LIMIT: 0,
```

Comment:
is `skip` and `limit` still a necessary return in this case?

Reply:
It's not super important, but it does signify that any skip and limit parameters that the user provided were ignored, and also is consistent with all the other return options

Reply:
👍